api/src/main/java/org/openmrs/module/fhir/api/impl/PatientServiceImpl.java

```diff
@@ -269,7 +270,16 @@ public void deletePatient(String id) {
 	
 	@Override
 	public Patient createFHIRPatient(Patient patient) {
-		org.openmrs.Patient omrsPatient = FHIRPatientUtil.generateOmrsPatient(patient);
+		List<String> errors = new ArrayList<String>();
+		org.openmrs.Patient omrsPatient = FHIRPatientUtil.generateOmrsPatient(patient, errors);
+		if (!errors.isEmpty()) {
+			StringBuilder errorMessage = new StringBuilder("The request cannot be processed due to the following issues \n");
```

Comment:
Mhhh I think since this is api level we may not get the internationalization in this level :(